cmd/kubernetesDeploy_test.go

```diff
@@ -57,16 +58,68 @@ func TestRunKubernetesDeploy(t *testing.T) {
 			"--force",
 			"--namespace",
 			"deploymentNamespace",
+			"--set",
```

Comment:
Moved this up, since we declare theses values before appending `--wait` and `--timeout` to the list of helm parameters.